circle.py

```diff
@@ -33,6 +33,31 @@ def generate_2D_coordinate(x_min, x_max, y_min, y_max):
     return (x, y)
 
 
-# Below is an example of how to generate a random 2D coordinate
-x, y = generate_2D_coordinate(-1, 1, -1, 1)
+'''
+ @brief	    Function that calcultes pi
+ @details   The function randomly places points in a 1x1 box using generate_2D_coordinates. It uses the ratio of points that land in a circle with radius one vs the number of points thrown to calculate pi.
+ @param     num_darts the number of points to be tested
+ @return    The calculated value for pi	
+
+
+'''
+
+def calculate_pi(num_darts):
+	hits = 0
+	misses = 0
+	for i in xrange(int(num_darts)):	
+		x, y = generate_2D_coordinate(-1, 1, -1, 1)
+		if math.sqrt(x**2 + y**2) <= 1:
+			hits += 1
```

Comment:
In general, I see you are using tabs instead of spaces. While this will work, it's generally not considered good style for a variety of reasons. Instead of using a tab, insert 4 spaces. This can be annoying, but you can set your editor settings to insert 4 spaces instead of a tab character when you hit the tab button.